docs/src/appendix/migrating-from-chiseltest.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Migrating from ChiselTest"
+sidebar_position: 0
+---
+
+# Migrating from ChiselTest to ChiselSim
+
+## Background
+
+With the release of Chisel 5, Chisel moved off of the legacy [Scala FIRRTL Compiler](https://github.com/chipsalliance/firrtl) to the MLIR FIRRTL Compiler (part of the [llvm/circt](https://github.com/llvm/circt) project).
+This change in underlying compiler technology has been a crucial piece in enabling the addition of many new features to Chisel, including linear-temporal logic (LTL) properties, Probes, and Layers.
+Unfortunately, Chisel 3's testing library, [ChiselTest](https://github.com/ucb-bar/chiseltest), is built around the SFC, making it difficult to support ChiselTest in Chisel 5 and beyond.
+
+This is not a fundamental problem, it is one of engineering resources.
+This page is intended to serve as a living document to help Chisel 3 users migrate from ChiselTest to Chisel's native simulation library, ChiselSim.
```

Comment:
Does ChiselSim exist? This was renamed to `svsim`, I'm not sure what ChiselSim is other than some specific implementations of svsim...?

Reply:
SVSim is the underlying execution engine, but the user-facing API is `chisel3.simulator` (ChiselSim). Also, the future arcilator will not work with SVSim (because SVSim is intentionally closely tied to simulating SystemVerilog), so we will need a different execution engine for it, but it should still be targeted by the higher-level "ChiselSim" API.